.github/workflows/test-and-release.yml

```diff
@@ -0,0 +1,27 @@
+name: test-and-release
+
+on: push
+
+concurrency:
+    group: ${{ github.workflow}}-${{ github.ref }}
+    # Cancel previous runs if not on a release branch
+    cancel-in-progress: ${{ !contains(github.ref, fromJSON('["master", "main"]')) }}
+
+jobs:
+    lint-commits:
+        uses: dhis2/workflows-platform/.github/workflows/lint-commits.yml@v1
+    lint:
+        uses: dhis2/workflows-platform/.github/workflows/lint.yml@v1
+    test:
+        uses: dhis2/workflows-platform/.github/workflows/test.yml@v1
+    e2e:
+        uses: dhis2/workflows-platform/.github/workflows/e2e.yml@v1
+        # Skips forks and dependabot PRs
+        if: '!github.event.push.repository.fork'
+        secrets: inherit
+    release:
+        needs: [lint, test, e2e]
```

Comment:
I know nothing has changed, but why does this depend on those 3 jobs, but not on lint-commits?